src/paulscode/android/mupen64plusae/GameLifecycleHandler.java

```diff
@@ -242,6 +255,284 @@ public void onCreateEnd( Bundle savedInstanceState )
         
         // Override the peripheral controllers' key provider, to add some extra functionality
         inputSource.setOnKeyListener( this );
+        
+        if( savedInstanceState == null )
+        {
+            // Show the drawer at the start and have it hide itself automatically
```

Comment:
This is entirely for usability and discoverability, to let the user know there's a slide-out drawer along the left edge. They have to use the sidebar to get to gameplay in the first place, so they should be familiar with the concept by this point.